fastify.js

```diff
@@ -518,15 +531,82 @@ function build (options) {
     socket.end(`HTTP/1.1 400 Bad Request\r\nContent-Length: ${body.length}\r\nContent-Type: 'application/json'\r\n\r\n${body}`)
   }
 
-  function defaultRoute (req, res, params) {
+  function defaultRoute (req, res) {
+    fourofour.lookup(req, res)
+  }
+
+  function basic404 (req, reply) {
+    reply.code(404).send(new Error('Not found'))
+  }
+
+  function fourFourFallBack (req, res) {
```

Comment:
This would be easier to recognize in the future if it is named `fourOhFourFallBack`.